designer/server/src/lib/formPersistenceService.js

```diff
@@ -0,0 +1,26 @@
+import Wreck from '@hapi/wreck'
+
+import config from '~/src/config.js'
+
+/**
+ * Get draft form definition
+ * @param {string} id
+ * @returns {Promise<import('@defra/forms-model').FormDefinition | null>}
+ */
+export async function getDraftFormDefinition(id) {
+  const { payload } = await Wreck.get(
+    `${config.managerUrl}/forms/${id}/definition/draft`
+  )
+  return payload
+}
```

Comment:
I think we should be using the new forms fetch library @davidjamesstone added

It's been written for `FormConfiguration` but the wreck payloads are typed in JSDoc

Shall we extend it for `FormDefinition` too?

```
import * as forms from '~/src/lib/forms.js'
```

Reply:
Similar to importing the S3 client from AWS, all of this code could live in **forms-manager** at some point

One for another chat